taichi/backends/metal/device.cpp

```diff
@@ -34,7 +34,8 @@ class ResourceBinderImpl : public ResourceBinder {
                  uint32_t binding,
                  DevicePtr ptr,
                  size_t size) override {
-    TI_NOT_IMPLEMENTED;
+    TI_ERROR(
```

Comment:
Is this a TODO or a not supported feature in Metal?

Reply:
TODO

Reply:
More specifically, Metal has this API where you can set `offset`, but not `size`... https://developer.apple.com/documentation/metal/mtlcomputecommandencoder/1443126-setbuffer?language=objc

Reply:
This is actually fine, we can ignore size.

Reply:
In vulkan when offset+size go over the buffer size it's undefined behavior anyways

Reply:
Fixed